### PR TITLE
Retrieving info from Database and Client

### PR DESCRIPTION
**Pull request by AnthonyOdum**
- Reorganized methods into different files
- Retrieves information from Database and Client

### CODE REVIEW

File: api/src/main/java/com/google/sps/SendNotifications.java [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+      DocumentSnapshot gameDocSnap = game.get().get();
+      System.out.println("Is this the error?...");
+      List<String> playerNames = (List<String>) gameDocSnap.get("players");
+      System.out.println(playerNames);
+
+      for(String player: playerNames) {
+
+        DocumentSnapshot usersDocSnap = players.document(player).get().get();
+
+        //Retrieves player information
```
Comment (mrdanding) [inline api/src/main/java/com/google/sps/SendNotifications.java:67]: Nit: an empty space before the comment here and below.

Like: // Retrieves....

File: api/src/main/java/com/google/sps/SendNotifications.java [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+        String playerEmail = usersDocSnap.getString("email");
+        String playerName = usersDocSnap.getString("username");
+
+        System.out.println(playerEmail);
+
+        //Adding player to Email object
+        emails.add(new Email(new User(playerEmail, playerName)));
+
+        //Adds Start game message to the email object
+        emails.get(emails.size()-1).playerTurn(gameID);
```
Comment (mrdanding) [inline api/src/main/java/com/google/sps/SendNotifications.java:77]: Do we need any check here?

File: api/src/main/java/com/google/sps/SendNotifications.java [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+    byte[] bytes = buffer.toByteArray();
+    String encodedEmail = Base64.encodeBase64URLSafeString(bytes);
+    Message message = new Message();
+    message.setRaw(encodedEmail);
+    return message;
+  }
+
+  private static Message sendMessage(Gmail service, String userId, MimeMessage emailContent)
+    throws MessagingException, IOException {
+  Message message = createMessageWithEmail(emailContent);
```
Comment (mrdanding) [inline api/src/main/java/com/google/sps/SendNotifications.java:143]: Nit: idenation.

File: api/src/main/java/com/google/sps/SendNotifications.java (hunk 2 above)
Comment (AnthonyOdum) [reply api/src/main/java/com/google/sps/SendNotifications.java:77]: there should always be one player in the game so i don't think it should be a problem

File: api/src/main/java/com/google/sps/SendNotifications.java (hunk 1 above)
Comment (AnthonyOdum) [reply api/src/main/java/com/google/sps/SendNotifications.java:67]: fixed!

File: api/src/main/java/com/google/sps/SendNotifications.java (hunk 3 above)
Comment (AnthonyOdum) [reply api/src/main/java/com/google/sps/SendNotifications.java:143]: fixed!

File: api/src/main/java/com/google/sps/Application.java [hunk 4]
```diff
@@ -5,78 +5,23 @@
...
-import java.util.ArrayList;
-import java.util.List;
-
 @SpringBootApplication
 @RestController
 
 public class Application {
-  @GetMapping("/notify")
-  public static void main(String[] args) {
+	public static void main(String[] args) {
```
Comment (tomatocat) [inline api/src/main/java/com/google/sps/Application.java:12]: fix indentation: convert tabs to spaces w/ ctrl+shift+p and search the function

**Pull request merged**